css/at-rules/font-feature-values.json

```diff
@@ -103,6 +103,42 @@
             }
           }
         },
+        "font-display": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@font-featur-values/font-display",
+            "spec_url": "https://drafts.csswg.org/css-fonts/#font-display-font-feature-values",
+            "support": {
+              "chrome": {
+                "version_added": "60"
```

Comment:
CI fails because Chrome 60 can't be correct if the parent feature is not supported. (`font-display` can't exist without `@font-feature-values`). 

So either change this to `false` or update `@font-feature-values` to say it is supported in Chrome.

Reply:
estide

Reply:
> CI fails because Chrome 60 can't be correct if the parent feature is not supported. (`font-display` can't exist without `@font-feature-values`). 
> 
> So either change this to `false` or update `@font-feature-values` to say it is supported in Chrome.

Reply:
> CI fails because Chrome 60 can't be correct if the parent feature is not supported. (`font-display` can't exist without `@font-feature-values`). 
> 
> So either change this to `false` or update `@font-feature-values` to say it is supported in Chrome.

Reply:
> CI fails because Chrome 60 can't be correct if the parent feature is not supported. (`font-display` can't exist without `@font-feature-values`). 
> 
> So either change this to `false` or update `@font-feature-values` to say it is supported in Chrome.

Reply:
> CI fails because Chrome 60 can't be correct if the parent feature is not supported. (`font-display` can't exist without `@font-feature-values`). 
> 
> So either change this to `false` or update `@font-feature-values` to say it is supported in Chrome.

Reply:
> CI fails because Chrome 60 can't be correct if the parent feature is not supported. (`font-display` can't exist without `@font-feature-values`). 
> 
> So either change this to `false` or update `@font-feature-values` to say it is supported in Chrome.

Reply:
> CI fails because Chrome 60 can't be correct if the parent feature is not supported. (`font-display` can't exist without `@font-feature-values`). 
> 
> So either change this to `false` or update `@font-feature-values` to say it is supported in Chrome.

Reply:
> estide
>

Reply:
> > CI fails because Chrome 60 can't be correct if the parent feature is not supported. (`font-display` can't exist without `@font-feature-values`). 
> > 
> > So either change this to `false` or update `@font-feature-values` to say it is supported in Chrome. 
> 
>

Reply:
> CI fails because Chrome 60 can't be correct if the parent feature is not supported. (`font-display` can't exist without `@font-feature-values`). 
> 
> So either change this to `false` or update `@font-feature-values` to say it is supported in Chrome. 

estelle